src/fee_rate.rs

```diff
@@ -0,0 +1,69 @@
+use super::*;
+
+#[derive(Debug, PartialEq, Clone, Copy)]
+pub(crate) struct FeeRate(u64);
```

Comment:
I don't exactly remember what we were thinking

```suggestion
pub(crate) struct FeeRate(f64);
```